docs/conf.py

```diff
@@ -0,0 +1,90 @@
+# -*- coding: utf-8 -*-
+
+import os
+import sys
+from pathlib import Path
+
+# -- Sphinx Options -----------------------------------------------------------
+
+# If your project needs a minimal Sphinx version, state it here.
+needs_sphinx = "3.0"
+
+extensions = [
+    "sphinx.ext.autodoc",
+    "sphinx.ext.extlinks",
+    "sphinx.ext.intersphinx",
+    "sphinx.ext.todo",
+    "sphinxarg.ext",  # Automatic argparse command line argument documentation
+]
+
+autodoc_default_options = {
+    "members": True,
+}
+
+# The suffix(es) of source filenames.
+source_suffix = {
+    ".rst": "restructuredtext",
+    # '.txt': 'markdown',
+    # '.md': 'markdown',
+}
+
+master_doc = "index"
+
+project = u"Introduction to TDD and CI With VUnit"
+copyright = u"2014-2021, Lars Asplund"
+author = u"LarsAsplund, kraigher and contributors"
+
+version = ""
+release = ""
+
+language = None
+
+exclude_patterns = []
+
+pygments_style = "sphinx"
+
+todo_include_todos = False
+
+# -- Options for HTML output ----------------------------------------------
+
+html_theme_path = ["."]
+html_theme = "_theme"
+
+html_theme_options = {
+    "analytics_id": "UA-abcdefghi-j",
+    "logo_only": True,
+    "vcs_pageview_mode": "blob",
+    "style_nav_header_background": "#0c479d",
+    "home_breadcrumbs": False,
+}
+html_context = {
+    "conf_py_path": "%s/" % Path(__file__).parent.name,
```

Comment:
When using the BTD Action, this is set automatically by adding the following snippet: 

- https://buildthedocs.github.io/btd/cli/themes.html
- For instance: https://github.com/edaa-org/edaa-org.github.io/blob/main/conf.py#L91-L94